person.rb

```diff
@@ -1,25 +1,31 @@
-class Person
+require_relative 'nameable'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
+class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
 
   def initialize(age, parent_permission: true, name: 'Unknown')
```

Comment:
- [ ] It would be better if we fix the bug so that the test can be run successfully without any errors. I created a new file `main.rb` and copied the contents of `person = Person.new(22, 'maximilianus')
 person.correct_name
 capitalizedPerson = CapitalizeDecorator.new(person)
 capitalizedPerson.correct_name
 capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
 capitalizedTrimmedPerson.correct_name
`
In it then i run `ruby main.rb` to test it. The screenshot below shows the result of the test
![Screenshot from 2023-06-28 20-46-28](https://github.com/Mikey1-nuel/school_library/assets/46388113/066ad121-a893-420d-92ec-e35a853b6b84)